4001-5000/LIT4633Chronicle.xml

```diff
@@ -880,8 +894,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 </div>
                 
                 <div type="textpart" subtype="paragraph" n="47">
-                    <ab>1ክፍል፡ ፲፪።2ሠረቀ፡ ሐምሌ፡ በዕለተ፡ ሠሉስ።3ወአመ፡ ፪፡ ለሐምሌ፡ ሞተ፡ ባሻ፡ ወልደ፡ ጊዮርጊስ፡ 
-                        ቈለኛ፡ ወተቀብረ፡ በደብረ፡ (G7v) ብርሃን።4ወበውእቱ፡ ወርኅ፡ ተስዕረ1፡ ባሻ፡ አርከሌድስ፡ ወተሠይመ፡ ባሻ፡ አደራ።
+                    <ab>ክፍል፡ ፲፪።2ሠረቀ፡ ሐምሌ፡ በዕለተ፡ ሠሉስ። ወአመ፡ ፪፡ ለሐምሌ፡ ሞተ፡ ባሻ፡ ወልደ፡ ጊዮርጊስ፡ 
+                        ቈለኛ፡ ወተቀብረ፡ በ<placeName ref="LOC2248DabraB">ደብረ፡ ብርሃን።4</placeName>ወበውእቱ፡ ወርኅ፡ ተስዕረ፡
```

Comment:
Does the 4 belong here?

Reply:
I will delete all Arabic numerals that are not part of the text, but they are footnotes number for the edition.